pkg/plugins/internal/machinery/scaffold.go

```diff
@@ -343,7 +343,7 @@ func insertStrings(content string, codeFragmentsMap file.CodeFragmentsMap) ([]by
 		line := scanner.Text()
 
 		for marker, codeFragments := range codeFragmentsMap {
-			if strings.TrimSpace(line) == strings.TrimSpace(marker.String()) {
+			if marker.EqualsLine(line) {
```

Comment:
if we added here || strings.TrimSpace(line) == strings.TrimSpace(marker.String()) would not solve for both scenarios?